### PR TITLE
Prevent double-escaping in JS templates by removing escaping here

### PR DESCRIPTION
**Pull request by danielbachhuber**
See #502, #456

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@goldenapples #reviewmerge

File: inc/class-shortcode-ui.php [hunk 1]
```diff
@@ -217,16 +217,16 @@ public function enqueue() {
...
-				'search_placeholder'                => esc_html__( 'Search', 'shortcode-ui' ),
-				'insert_content_label'              => esc_html__( 'Insert Content', 'shortcode-ui' ),
+				'media_frame_title'                 => __( 'Insert Post Element', 'shortcode-ui' ),
+				'media_frame_menu_insert_label'     => __( 'Insert Post Element', 'shortcode-ui' ),
+				'media_frame_menu_update_label'     => __( '%s Details', 'shortcode-ui' ), // Substituted in JS
+				'media_frame_toolbar_insert_label'  => __( 'Insert Element', 'shortcode-ui' ),
+				'media_frame_toolbar_update_label'  => __( 'Update', 'shortcode-ui' ),
+				'media_frame_no_attributes_message' => __( 'There are no attributes to configure for this Post Element.', 'shortcode-ui' ),
+				'edit_tab_label'                    => __( 'Edit', 'shortcode-ui' ),
+				'mce_view_error'                    => __( 'Failed to load preview', 'shortcode-ui' ),
```
Comment (goldenapples) [inline inc/class-shortcode-ui.php:226]: This [isn't being escaped on output](https://github.com/fusioneng/Shortcake/blob/master/js/src/utils/shortcode-view-constructor.js#L26).

Comment (danielbachhuber) [inline inc/class-shortcode-ui.php:226]: Ok. I'll update so we're properly building those strings.

File: inc/class-shortcode-ui.php [hunk 2]
```diff
@@ -217,16 +217,16 @@ public function enqueue() {
...
+				'search_placeholder'                => __( 'Search', 'shortcode-ui' ),
```
Comment (goldenapples) [inline inc/class-shortcode-ui.php:227]: Neither is [this](https://github.com/fusioneng/Shortcake/blob/master/js/src/views/search-shortcode.js#L15), from what I can tell.

File: inc/class-shortcode-ui.php [hunk 3]
```diff
@@ -217,16 +217,16 @@ public function enqueue() {
...
-				'mce_view_error'                    => esc_html__( 'Failed to load preview', 'shortcode-ui' ),
-				'search_placeholder'                => esc_html__( 'Search', 'shortcode-ui' ),
-				'insert_content_label'              => esc_html__( 'Insert Content', 'shortcode-ui' ),
+				'media_frame_title'                 => __( 'Insert Post Element', 'shortcode-ui' ),
+				'media_frame_menu_insert_label'     => __( 'Insert Post Element', 'shortcode-ui' ),
+				'media_frame_menu_update_label'     => __( '%s Details', 'shortcode-ui' ), // Substituted in JS
+				'media_frame_toolbar_insert_label'  => __( 'Insert Element', 'shortcode-ui' ),
+				'media_frame_toolbar_update_label'  => __( 'Update', 'shortcode-ui' ),
+				'media_frame_no_attributes_message' => __( 'There are no attributes to configure for this Post Element.', 'shortcode-ui' ),
+				'edit_tab_label'                    => __( 'Edit', 'shortcode-ui' ),
```
Comment (goldenapples) [inline inc/class-shortcode-ui.php]: This is not being used anywhere, AFAICT...

File: inc/class-shortcode-ui.php (hunk 2 above)
Comment (goldenapples) [inline inc/class-shortcode-ui.php:227]: Never mind, it's being escaped in `wp.media.view.Search`.

**Comment by goldenapples** [comment]
Most of the strings we're using are being properly built using js templates or with `$.text()` or similar. I just saw one cases that wasn't, which should be updated, and one string which isn't being used at all, which can be removed.

**Comment by danielbachhuber** [comment]
@goldenapples ready to :ship:

**Comment by goldenapples** [comment]
Looks good to me.

**Pull request merged**